protected/humhub/modules/user/config.php

```diff
@@ -6,13 +6,15 @@
 use humhub\modules\content\components\ContentAddonActiveRecord;
 use humhub\modules\content\components\ContentActiveRecord;
 use humhub\commands\CronController;
+use humhub\widgets\TopMenu;
 
 return [
     'id' => 'user',
     'class' => \humhub\modules\user\Module::class,
     'isCoreModule' => true,
     'urlManagerRules' => [
-        ['class' => 'humhub\modules\user\components\UrlRule']
+        ['class' => 'humhub\modules\user\components\UrlRule'],
+        'people' => 'user/people',
```

Comment:
`/people` is suffient

Reply:
Same behaviour I described in https://github.com/humhub/humhub/pull/5057#discussion_r644769989